build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -34,79 +30,78 @@ module.exports = function ({types: t}) {
     }
   }
 
+  function joinTemplateLiterals(left, right) {
+    // First merge the first member of the right quasi into the last quasi on the left.
+    const fromQuasi = right.get('quasis.0');
+    const toQuasi = left.get(`quasis.${left.node.quasis.length - 1}`);
+    toQuasi.node.value.raw += fromQuasi.node.value.raw;
+    toQuasi.node.value.cooked += fromQuasi.node.value.cooked;
+
+    // Can safely remove the merged quasi.
+    fromQuasi.remove();
+
+    // Merge the right remaining quasis and expressions to ensure merged left is valid.
+    left.node.quasis.push(...right.node.quasis);
+    left.node.expressions.push(...right.node.expressions);
+
+    // Now the left contains the values of the right, so remove the right.
+    right.remove();
+  }
+
+  function joinMaybeTemplateLiteral(path) {
+    const left = path.get('left');
+    const right = path.get('right');
+    if (left.isTemplateLiteral()) {
+      if (right.isTemplateLiteral()) {
+        // When both sides are template literals, bypass `babel.evaluate` since it cannot handle this condition.
+        joinTemplateLiterals(left, right);
+        return;
+      }
+
+      const e = right.evaluate();
+      if (e.confident) {
+        const quasi = left.node.quasis[left.node.quasis.length - 1];
+        quasi.value.raw += escapeForLiteral(e.value);
+        quasi.value.cooked += escapeForLiteral(e.value);
```

Comment:
You don't want to escaped the cooked value, only raw.

Reply:
Should be resolved with 95f48f1.